main.py

```diff
@@ -149,11 +151,16 @@ def file_paths_to_dict(resume_file: Path | None, plain_text_resume_file: Path) -
 
         return result
 
-def init_browser() -> webdriver.Chrome:
+def init_browser(browser: str = 'chrome') -> webdriver.Chrome | webdriver.Firefox:
     try:
-        options = chrome_browser_options()
-        service = ChromeService(ChromeDriverManager().install())
-        return webdriver.Chrome(service=service, options=options)
+        if browser == 'chrome':
```

Comment:
can you use factory pattern for this

Reply:
Yeah, I have created a browser factory class in `src/webdrivers/browser_factory.py`